lib/util.js

```diff
@@ -71,11 +71,17 @@ function parseHeaders (headers, obj) {
   return obj
 }
 
+function isBuffer (buffer) {
+  // See, https://github.com/mcollina/undici/pull/319
+  return buffer instanceof Uint8Array || Buffer.isBuffer(buffer)
```

Comment:
I'm totally lost on why Buffer.isBuffer would work, but hey.

Reply:
Related? https://github.com/facebook/jest/issues/9983

Reply:
https://github.com/facebook/jest/issues/9993

Reply:
https://github.com/facebook/jest/blob/4bd3d4a05999170f423f7050d4e0537648499e88/packages/jest-environment-jsdom/src/index.ts#L49

Reply:
I'm also not sure, as the `isBuffer` implementation uses an `instanceof` as well: https://github.com/nodejs/node/blob/7a447bcd54f5e153d14da377963541349d7bda09/lib/buffer.js#L504
But yeah the new test fails without the fix and works with the fix.

Reply:
Shouldn't it be enough to do instanceof Uint8Array?
Buffer is a instanceof Uint8Array after all...